source/fuzz/repeated_pass_instances.h

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SOURCE_FUZZ_REPEATED_REPEATED_PASS_INSTANCES_
```

Comment:
Not a big issue but don't forget to add `H_` at the end -> `SOURCE_FUZZ_REPEATED_REPEATED_PASS_INSTANCES_H_`.

Reply:
The same for other header files.

Reply:
Fixed.